### ISSUE TITLE
first voice prompt is delayed

### ISSUE THREAD

**Issue by kaicarver** [AlexGilleran/HIITMe#4]
when I start running a program, the very first voice prompt is uttered with a delay, usually 5 seconds or a bit more.

if I stop and restart the program soon after, sometimes there is not that delay.

this is on an old Sony C3

It's a minor bug. I am just starting to use this app, which seems quite nice to me, as I was thinking of writing my own. I realize the author has given up on Android, sorry to hear that, warning duly noted! Thanks anyway for this app.

**Comment by AlexGilleran** [comment]
Yeah I seem to recall it's because it takes the speech system a while to start up - I think I tried mitigating this by initializing the speech system right at app startup but if you don't take long to select a program and hit the play button there's still a delay.

**Comment by kaicarver** [comment]
Hi Alex, thanks for your reply! I may try fixing that at some point, but it's no big deal, and my phone's slowness causes all sorts of weird things in general. 

Unrelated to this issue, but since I know I can reach you this way, I wanted to let you know I did a quick-and-dirty clone of your app, added a hard-wired 12-step "learn to run" program, and published to the Play Store, link below. I did this so I could quickly and easily share that program with friends, without needing to laboriously enter all the steps.

Let me know if this is not OK with you, in which case I will delete my cloned version of the app from the app store. I probably did other stuff wrong, so let me know if you see anything egregious. I left your name in the package name since my change is so small. If I make bigger changes I guess I should rename it  from alexgilleran.* to kaicarver.* or something.

In any case thanks for your app, it's been fun playing with it. (Sort of!... Android app development does feel like a bit of a pain...).

https://github.com/kaicarver/HIITMe
https://play.google.com/store/apps/details?id=com.alexgilleran.hiitmeS1S12

**Comment by AlexGilleran** [comment]
No problems from me as long as it stays open source, good to see it get used 🙂
